dist/znk.infra.js

```diff
@@ -2920,11 +2920,14 @@ angular.module('znk.infra.general')
                 this.setter = function(path, newVal){
                     return $q.when(entitySetter(path,newVal));
                 };
+
+                this.entityCache = $cacheFactory('entityCache' + cacheId);
```

Comment:
here you use angular's $cacheFacory ? maybe we need the ones that in angular-cahce ? we used it in the act qotd since it offers more functionality like get all keys and more